docs/data-sources/resource.md

```diff
@@ -0,0 +1,61 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_resource Data Source - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Resources in Twingate represent servers on the private network that clients can connect to. Resources can be defined by IP, CIDR range, FQDN, or DNS zone. For more information, see the Twingate documentation https://docs.twingate.com/docs/resources-and-access-nodes.
+---
+
+# twingate_resource (Data Source)
+
+Resources in Twingate represent servers on the private network that clients can connect to. Resources can be defined by IP, CIDR range, FQDN, or DNS zone. For more information, see the Twingate [documentation](https://docs.twingate.com/docs/resources-and-access-nodes).
+
+## Example Usage
+
+```terraform
+data "twingate_resource" "foo" {
+  id = "<your resource's id>"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `id` (String) The ID of the Resource
+
+### Read-Only
+
+- `address` (String) The Resource's IP/CIDR or FQDN/DNS zone
```

Comment:
Change to:

`The Resource's address, which may be an IP address, CIDR range, or DNS address`

Reply:
updated